cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/DistributedProgramRuntimeService.java

```diff
@@ -349,7 +349,7 @@ public ProgramLiveInfo getLiveInfo(Id.Program program, ProgramType type) {
     public ClusterResourceReporter(MetricsCollectionService metricsCollectionService, Configuration hConf,
                                    CConfiguration cConf) {
       super(metricsCollectionService.getCollector(
-        ImmutableMap.of(Constants.Metrics.Tag.NAMESPACE, Constants.SYSTEM_NAMESPACE)));
```

Comment:
shouldn't the overriding of namespace tag below be enough? we still want to specify system namespace for other stats. Or you rely on defaulting to system namespace on back-end?